config/_stage_run_pytest.sh

```diff
@@ -0,0 +1,42 @@
+set -x
+
+source venv/bin/activate
+export PYTHONPATH=$(pwd):$PYTHONPATH
+
+PR_NAME="$1"
+PR_AUTHOR="$2"
+LAB_PATH="$3"
+PYTEST_LABEL="$4"
+
+python config/skip_check.py --pr_name "$PR_NAME" --pr_author "$PR_AUTHOR" --lab_path "$LAB_PATH"
+
+if [ $? -eq 0 ]; then
+  echo 'skip check due to special conditions...' && exit 0
+fi
+
+TARGET_SCORE=$(bash config/get_mark.sh "$LAB_PATH")
+
+if [[ "$LAB_PATH" == "lab_5_scrapper" ]]; then
+  python config/config_param_changer.py --config_path="lab_5_scrapper/scrapper_config.json"
+  echo "Changed config params"
+fi
+
+if [[ "$LAB_PATH" == "lab_5_scrapper" ]]; then
+  python -m pytest -m "mark${TARGET_SCORE} and ${PYTEST_LABEL}" --capture=no --ignore=lab_6_pipeline
+else
+  if [[ ${TARGET_SCORE} != 10 ]]; then
+    python -m pytest -m "mark${TARGET_SCORE} and ${PYTEST_LABEL}" --capture=no --ignore lab_6_pipeline/tests/s4_pos_frequency_pipeline_test.py --ignore lab_6_pipeline/tests/s3_6_advanced_pipeline_test.py
```

Comment:
why do we need ignoring of particular files?

Reply:
fixed